asg.tf

```diff
@@ -9,15 +9,27 @@ set -e
   EOF
 
   user_data_tail = <<EOF
+currentArch=$(uname -m)
```

Comment:
fun fact. `arch` variable (function) is [already taken](https://www.geeksforgeeks.org/arch-command-in-linux-with-examples/) in the Unix systems.

I didn't want to use it though, I trust `uname -m` a bit better.